### PR TITLE
[capz]: add devigned (david justice) to reviewers

### PR DESCRIPTION
**Pull request by devigned**
Adding myself as a reviewer for Cluster API Provider Azure.

/cc @CecileRobertMichon, @justaugustus

### CODE REVIEW

**Comment by CecileRobertMichon** [comment]
/hold

actually... this needs to be alphabetized

File: config/jobs/kubernetes-sigs/cluster-api-provider-azure/OWNERS [hunk 1]
```diff
@@ -13,6 +13,7 @@ reviewers:
 - juan-lee
 - serbrech
 - vincepri
+- devigned
```
Comment (CecileRobertMichon) [inline config/jobs/kubernetes-sigs/cluster-api-provider-azure/OWNERS:16]:
```suggestion
- devigned
- juan-lee
- serbrech
- vincepri
```

Comment (devigned) [reply config/jobs/kubernetes-sigs/cluster-api-provider-azure/OWNERS:16]: Haha! I was thinking about doing this, but I though it was not very nice to drop the other names down an index.

**Comment by justaugustus** [comment]
Eeep, I was supposed to get to this :'(
Sorry for the wait!
/lgtm
/approve

**Comment by justaugustus** [comment]
@devigned -- You need to become a member of the `kubernetes` org before this will merge.
Here's the issue template: https://github.com/kubernetes/org/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.md&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E

/hold

**Comment by CecileRobertMichon** [comment]
@devigned while you're at it there's also https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-cluster-api-azure/OWNERS

**Comment by CecileRobertMichon** [comment]
@justaugustus good catch, thanks!

**Pull request merged**